cvpy/biomedimage/BiomedImage.py

```diff
@@ -241,3 +242,62 @@ def rename_columns():
 
         # Change column names in the ImageTable
         rename_columns()
+
+    def morphological_gradient(self, images: ImageTable, kernel_width: int = 3,
+                               kernel_height: int = 3, copy_vars: List[str] = None) -> ImageTable:
+        """
+        Compute morphological gradient for given 3D grayscale image table.
```

Comment:
Does this compute one morphological gradient per image in the input table? If so, I might rephrase it as "Compute the morphological gradient for each image in the image table."

Reply:
How about  "Compute the morphological gradient for each 3D grayscale image in the image table."?

Reply:
Sure. That sounds great!